integration/kubernetes/runtimeclass_workloads/lifecycle-events.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: handlers
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: handlers-container
+    image: nginx
+    lifecycle:
+      postStart:
+        exec:
+          command: ["/bin/sh", "-c", "echo Hello from the postStart handler > /usr/share/message"]
```

Comment:
ooi, why `/usr/share/` ?

Reply:
Oh right. In that case, maybe keep it as close to those original files as possible then. In fact, I'd mention this fact in the commit (ideally also mentioning the commit version of the files you modified).